app/src/main/java/com/asfoundation/wallet/wallet_validation/generic/CodeValidationFragment.kt

```diff
@@ -58,22 +58,20 @@ class CodeValidationFragment : DaggerFragment(),
   }
 
```

Comment:
- [ ] countryCode codestyle should be improved

- [ ] phoneNumber codestyle should be improved

- [ ] errorMessage codestyle should be improved

- [ ] validationInfo codestyle should be improved